open_schools_platform/user_management/users/serializers.py

```diff
@@ -60,8 +59,15 @@ def create(self, validated_data):
         pass
 
 
-class UserSerializer(serializers.ModelSerializer):
+class HistoryRecordsField(serializers.ListField):
+    child = serializers.DictField()
```

Comment:
what is it?

Reply:
After calling the method .values() we need to check that data is correct